client/wfprev-war/src/main/angular/src/app/components/confirmation-dialog/confirmation-dialog.component.ts

```diff
@@ -34,7 +34,7 @@ export class ConfirmationDialogComponent {
   }
 
   get isDeleteDialog(): boolean {
```

Comment:
I'd recommend to change this function name to isDeleteOrChangeDialog to reflect the functionality

Reply:
ok done